.github/workflows/test.yml

```diff
@@ -131,5 +97,7 @@ jobs:
       env:
         PYOPENCL_COMPILER_OUTPUT: 1
       run: |
+        # make sure the test will fail if the gui test doesn't work
+        set -e -o pipefail
         cd src/sas/qtgui
-        xvfb-run -a --server-args="-screen 0 1024x768x24" python GUITests.py || true
```

Comment:
These test failures are suppressed with `|| true` because they fail badly. They are worth running because they're fast enough and to track the (very slow) progress towards fixing them.

Reply:
In order to track progress towards fixing them wouldn't it be better to mark them as known fails?

Reply:
That would be sensible too. I'm not sure how `@unittest.expectedFailure` interacts with the custom test collection and runner that is used in `GUITests.py`. (It's also not clear to me that there is any real value in the custom runner over what can be achieved fairly easily with pytest.)